renku/core/management/config.py

```diff
@@ -69,29 +80,40 @@ def local_config_path(self):
     @property
     def global_config_lock(self):
         """Create a user-level config lock."""
-        lock_file = "{0}/{1}.lock".format(self.global_config_dir, self.CONFIG_NAME)
+        lock_file = f"{self.global_config_path}.lock"
 
         import logging
 
         logging.getLogger("filelock").setLevel(logging.ERROR)
 
         return filelock.FileLock(lock_file, timeout=0)
 
-    def load_config(self, local_only, global_only):
+    def load_config(self, config_filter=ConfigFilter.ALL):
         """Loads local, global or both configuration object."""
         config = configparser.ConfigParser()
-        if local_only:
-            config_files = [self.local_config_path]
-        elif global_only:
-            config_files = [self.global_config_path]
-        else:
-            config_files = [self.global_config_path, self.local_config_path]
+        config_files = [resource_filename("renku", "data/defaults.ini")]
+
+        if config_filter == ConfigFilter.LOCAL_ONLY:
+            config_files += [self.local_config_path]
+        elif config_filter == ConfigFilter.GLOBAL_ONLY:
+            config_files += [self.global_config_path]
+        elif config_filter == ConfigFilter.ALL:
+            config_files += [self.global_config_path, self.local_config_path]
 
-        if not local_only:
+        if config_filter != ConfigFilter.LOCAL_ONLY:
             with self.global_config_lock:
                 config.read(config_files)
         else:
             config.read(config_files)
+
+        # transform config section for backwards compatibility
+        for s in config.sections():
+            if not s.startswith('renku "'):
```

Comment:
Does black not complain here? :thinking:

Reply:
About the `s` ?

Reply:
No I mean for the single quotes :D

Reply:
Actually, black is the one that changes it from `"renku \""` to `'renku "'` :man_shrugging: